frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap

```diff
@@ -223,7 +223,6 @@ Array [
      
     Ⓝ
   </span>,
-  " ",
```

Comment:
Without this space the phrase will be jammed together

Reply:
fixed now